grimshot: add notify action to open file or directory

willnorris opened the pull request:
If grimshot is called with --notify and the file was saved to disk, provide two notification actions to open the file or the containing directory. Use xdg-open to open the file/directory, so also add that to the check command.

OctopusET commented on grimshot:
```diff
@@ -78,7 +78,16 @@ if [ "$ACTION" != "save" ] && [ "$ACTION" != "copy" ] && [ "$ACTION" != "savecop
+    notify-send -t 3000 -a grimshot "$@"
+  else
+    action=$(notify-send -t 3000 -a grimshot --action=file='Open File' --action=dir='Open Directory' "$@")
+    if [ "$action" = "file" ]; then
+      xdg-open "$FILE"
+    elif [ "$action" = "dir" ]; then
+      xdg-open "$(dirname "$FILE")"
+    fi
```
```suggestion
    if [ "$action" = "dir" ]; then
      xdg-open "$(dirname "$FILE")"
    else
      xdg-open "$FILE"
    fi
```
With notification programs like `mako`, you can't select the action. So make opening file as default would be better.

OctopusET replied:
Oh this suggestion makes it always open the file, ignore that.

OctopusET replied:
Do you have an idea for this?

willnorris replied:
finally getting back to this.  I think for notification programs like mako that don't support actions, we simply don't take any actions.  So for them, this change is a noop.  I tested with mako and it seems that is the current behavior as written, so I don't think any changes would be necessary for that?

willnorris committed:
grimshot: add notify action to open file or directory

If grimshot is called with --notify and the file was saved to disk,
provide two notification actions to open the file or the containing
directory. Use xdg-open to open the file/directory, so also add that to
the check command.
